src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue17782.cs

```diff
@@ -0,0 +1,29 @@
+#if !MACCATALYST
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue17782 : _IssuesUITest
+{
+	public Issue17782(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "[ManualMauiTests] New text in the Editor character spacing test sometimes uses the previous spacing";
+
+	[Test]
+	[Category(UITestCategories.Editor)]
+	public void VerifyEditorCharacterSpacingWithText()
+	{
+		App.WaitForElement("SliderCharacterSpacing");
+		App.Click("ButtonAddEditorText");
+		App.Click("ButtonUpdateCharacterSpacing");
+		App.EnterText("DynamicCharacterSpacingEditor", "Text");
+		App.Click("ButtonResetCharacterSpacing");
+		App.EnterText("ResetCharacterSpacingEditor", "Text");
+		VerifyScreenshot();
```

Comment:
The test is failing on iOS. Next, a screenshot where can see the differences (red):
![image](https://github.com/user-attachments/assets/f79dffe6-79de-4d6b-96c1-4b7e559a8291)


Can use the `App.DismissKeyboard()` method to close the virtual keyboard if want to avoid it in the snapshot.

Reply:
Thank you for the feedback. We observed successful image generation during our testing. The discrepancies might stem from version differences between the emulator and simulator environments in CI.As per your suggestion added App.DismissKeyboard().